File: CS/WebSite/App_Code/Address.cs
using System;
using DevExpress.Xpo;

 
	
	public class Address : XPObject {
		public Address(Session session) : base(session) { }

        string city;
        public string City {
            get { return city; }
            set { SetPropertyValue("City", ref city, value); }
        }
    }

File: VB/WebSite/App_Code/Address.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo



	Public Class Address
		Inherits XPObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub

		Private city_Renamed As String
		Public Property City() As String
			Get
				Return city_Renamed
			End Get
			Set(ByVal value As String)
				SetPropertyValue("City", city_Renamed, value)
			End Set
		End Property
	End Class

